global.json

```diff
@@ -1,7 +1,6 @@
 {
     "sdk": {
-        // Keep the SDK (Major and Minor) version in sync with the .NET runtime version used in the Dockerfile
-        "version": "9.0.300",
-        "rollForward": "feature"
+        // Keep this version in sync with the .NET SDK version used in the Dockerfile
```

Comment:
I wonder about SDK vs Runtime version. I recently added a test to enforce same major & minor version. That test will fail when this is merged.

Reply:
I changed that test inhttps://github.com/Azure/azure-mcp/pull/432/commits/42b6d3ac2113bacacb224e75a52fc0812ec807a9 to track what I feel is the correct invariant - that the SDK used for building is _at least_ the version of the dockerfile's runtime. .NET SDKs can _always_ target/build for earlier runtimes, so keeping them exactly in sync is not required.

Reply:
Great, Do you want to update the comment here? Perhaps it can be removed entirely.